.github/workflows/pull-request.yml

```diff
@@ -490,3 +490,12 @@ jobs:
       - name: workflow success
         run: exit 0
         if: ${{ contains(needs.*.result, 'success') }}
+      - uses: slackapi/slack-github-action@v1.23.0
+        name: Nofify QA hxp-studio-testing failure
+        if: ${{ contains(needs.*.result, 'failure') }}
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+        with:
+          channel-id: 'C016SMNNL8L' #guild-channel
```

Comment:
should this somehow also notify A/N BDU?

Reply:
if you are in the guild front end (and you are) channel you will be notified